time_tracker_api/users/users_namespace.py

```diff
@@ -8,8 +8,22 @@
 
 ns = api.namespace('users', description='Namespace of the API for users')
 
-# User Model
+# Data to check if a user is in the group
+data_is_user_in_group = ns.model(
```

Comment:
I think we can rename it to 'user_in_group_input'

Reply:
I also think this model should be defined after the model 'User'

Reply:
Also, I think it is a good idea, also define a `UserInGroupResponse`  model. To add it with `@ns.marshal_with` decorators and improve swagger docs.

Reply:
Yes, i think it's a good way to improve swagger docs, i already put them.